src/misclibs.F

```diff
@@ -4188,45 +4028,25 @@ subroutine totq(nstat,rstat,ruh,rvh,rmh,q,rho,aname)
 
       integer i,j,k
       double precision :: tmass,var
-      double precision, dimension(nk) :: foo
+      !double precision, dimension(nk) :: foo
       double precision :: foot
 
-#ifdef _B4B08R
-      !$acc update host(rho,q,ruh,rvh,rmh)
-#else
-      !$acc data create(foo)
 
+      tmass=0.0d0
       !$omp parallel do default(shared) private(i,j,k)
-      !$acc parallel default(present) private(i,j,k) reduction(+:foot)
+      !$acc parallel default(present) private(i,j,k) reduction(+:foot,tmass)
       !$acc loop gang
```

Comment:
Do we need to add `reduction(+:tmass)` here?

Reply:
Yes we do.